### PR TITLE
Fixing the arrows in rtl languages

### PR DESCRIPTION
**Pull request by sarahmarciano**
When using this datepicker component with rtl languages the two arrows for moving from month to month are inverted
so I've just added a little check to check if the locale is an rtl language (hope I've included all of them feel free to add more if you know more rtl languages) and then decide if the arrow shall increase the month or decrease it.
I've also changed the functions names since each function is rendering the right/left arrow and not the prev/next month.
see my comment on this issue: https://github.com/Hacker0x01/react-datepicker/issues/1271

### CODE REVIEW

File: src/calendar.jsx [hunk 1]
```diff
@@ -303,7 +303,12 @@ export default class Calendar extends React.Component {
       "react-datepicker__navigation--previous"
     ];
 
-    let clickHandler = this.decreaseMonth;
+    let clickHandler;
+	if(this.props.locale === 'he' || this.props.locale === 'ar') {
```
Comment (rafeememon) [inline src/calendar.jsx]: Instead of hardcoding these I think we should do some kind of detection.  Does the ltr/rtl flag exist in moment somewhere?

Comment (sarahmarciano) [reply src/calendar.jsx]: I also tried to find in moment.js a function that detect if a locale is rtl or ltr but I didn't find
instead you can maybe use this library: https://github.com/shadiabuhilal/rtl-detect
or build your own util function and have something like this: https://stackoverflow.com/questions/4192382/is-there-any-way-to-detect-an-rtl-language-in-java

Comment (rafeememon) [reply src/calendar.jsx]: Maybe we can steal the list of locales from https://github.com/shadiabuhilal/rtl-detect/blob/master/lib/rtl-detect.js#L137.  I'd also pull that into a util file and encapsulate it as `isLocaleRtl`

**Comment by martijnrusschen** [comment]
Build seems to be failing. Also, any way to add a test for this behaviour?

**Comment by martijnrusschen** [comment]
Closing due inactvity

**Pull request closed without merging**